homeassistant/components/zwave_js/discovery.py

```diff
@@ -530,6 +530,63 @@ class ZWaveDiscoverySchema:
         primary_value=SWITCH_BINARY_CURRENT_VALUE_SCHEMA,
         assumed_state=True,
     ),
+    # Heatit Z-TRM6
+    ZWaveDiscoverySchema(
+        platform=Platform.CLIMATE,
+        hint="dynamic_current_temp",
+        manufacturer_id={0x019B},
+        product_id={0x3001},
+        product_type={0x0030},
+        primary_value=ZWaveValueDiscoverySchema(
+            command_class={CommandClass.THERMOSTAT_MODE},
+            property={THERMOSTAT_MODE_PROPERTY},
+            type={ValueType.NUMBER},
+        ),
+        data_template=DynamicCurrentTempClimateDataTemplate(
+            lookup_table={
+                # Floor sensor
+                "Floor": ZwaveValueID(
+                    property_=THERMOSTAT_CURRENT_TEMP_PROPERTY,
+                    command_class=CommandClass.SENSOR_MULTILEVEL,
+                    endpoint=4,
+                ),
+                # Internal sensor
+                "Internal": ZwaveValueID(
+                    property_=THERMOSTAT_CURRENT_TEMP_PROPERTY,
+                    command_class=CommandClass.SENSOR_MULTILEVEL,
+                    endpoint=2,
+                ),
+                # Internal with limit by floor sensor
+                "Internal with floor limit": ZwaveValueID(
+                    property_=THERMOSTAT_CURRENT_TEMP_PROPERTY,
+                    command_class=CommandClass.SENSOR_MULTILEVEL,
+                    endpoint=2,
+                ),
+                # External sensor (connected to device)
+                "External": ZwaveValueID(
+                    property_=THERMOSTAT_CURRENT_TEMP_PROPERTY,
+                    command_class=CommandClass.SENSOR_MULTILEVEL,
+                    endpoint=3,
+                ),
+                # External sensor (connected to device) with limit by floor sensor (2x sensors)
+                "External with floor limit": ZwaveValueID(
+                    property_=THERMOSTAT_CURRENT_TEMP_PROPERTY,
+                    command_class=CommandClass.SENSOR_MULTILEVEL,
+                    endpoint=3,
+                ),
+                # This mode is a hack, changing climate entity does not change operating mode  - Use Internal temp.
```

Comment:
Please break long comments and strings around max 88 characters per line.

Reply:
What do we mean with "this mode is a hack"?

Reply:
Apparently the z-wave spec doesnt allow what is called Power regulator mode.
So when the thermostat operates in this mode changing the climate entity actually does nothing.

You controll how long thermostat by  adjusting parameter 25, this can be exposed as a slider in HA. 
This slider goes from 1-10, this sets the percent of an hour that the heating is active.

From the manual
```
Set the % of time the relay
should be active when using
PWER mode. (30-minute
duty cycle)

Range: 
1 - 10
10 to 100%.
20% (Default)

Reply:
Please clarify the comment a bit more, explaining that the comment is more like a side note about the device. These values represent the current temperature, so the ability to control the device isn't really relevant to the code at this place. We can keep the comment to explain a bit more what power regulator mode means, but we shouldn't make it more confusing for the reader than without the comment.

Reply:
@MartinHjelmare Have added a longer description here, explains the mode, limits and why use air sensor.
Seems better?

Reply:
@MartinHjelmare the changes seems fine and have been commited.